applications/browser/package.json

```diff
@@ -0,0 +1,108 @@
+        "node": ">=12.14.1 <13"
+    },
+    "theia": {
+        "frontend": {
+            "config": {
+                "applicationName": "CDT Cloud Blueprint",
+                "preferences": {
+                    "files.enableTrash": false
+                }
+            }
+        }
+    },
+    "dependencies": {
+        "@theia/bulk-edit": "1.22.1",
+        "@theia/callhierarchy": "1.22.1",
+        "@theia/console": "1.22.1",
+        "@theia/core": "1.22.1",
+        "@theia/debug": "1.22.1",
+        "@theia/editor": "1.22.1",
+        "@theia/editor-preview": "1.22.1",
+        "@theia/electron": "1.22.1",
+        "@theia/external-terminal": "1.22.1",
+        "@theia/file-search": "1.22.1",
+        "@theia/filesystem": "1.22.1",
+        "@theia/getting-started": "1.22.1",
+        "@theia/keymaps": "1.22.1",
+        "@theia/markers": "1.22.1",
+        "@theia/messages": "1.22.1",
+        "@theia/metrics": "1.22.1",
+        "@theia/mini-browser": "1.22.1",
+        "@theia/monaco": "1.22.1",
+        "@theia/navigator": "1.22.1",
+        "@theia/outline-view": "1.22.1",
+        "@theia/output": "1.22.1",
+        "@theia/plugin-dev": "1.22.1",
+        "@theia/plugin-ext": "1.22.1",
+        "@theia/plugin-ext-vscode": "1.22.1",
+        "@theia/preferences": "1.22.1",
+        "@theia/preview": "1.22.1",
+        "@theia/process": "1.22.1",
+        "@theia/property-view": "1.22.1",
+        "@theia/scm": "1.22.1",
+        "@theia/scm-extra": "1.22.1",
+        "@theia/search-in-workspace": "1.22.1",
+        "@theia/task": "1.22.1",
+        "@theia/terminal": "1.22.1",
+        "@theia/timeline": "1.22.1",
+        "@theia/typehierarchy": "1.22.1",
+        "@theia/userstorage": "1.22.1",
+        "@theia/variable-resolver": "1.22.1",
+        "@theia/vsx-registry": "1.22.1",
+        "@theia/workspace": "1.22.1",
+        "fs-extra": "^9.0.1",
+        "cdt-cloud-blueprint-product": "1.22.1",
+        "@theia/cpp-debug": "1.19.0",
+        "theia-traceviewer": "0.2.0-next.61895a7.0"
+    },
+    "devDependencies": {
+        "@theia/cli": "1.22.1"
+    },
+    "scripts": {
+        "clean": "theia clean",
+        "build": "yarn -s compile && yarn -s bundle",
+        "bundle": "theia build --mode development",
+        "compile": "tsc -b",
+        "rebuild": "theia rebuild:browser --cacheRoot ../..",
+        "start": "yarn -s rebuild && theia start --plugins=local-dir:./plugins",
+        "start:debug": "yarn -s start --log-level=debug",
+        "start:watch": "concurrently --kill-others -n tsc,bundle,run -c red,yellow,green \"tsc -b -w --preserveWatchOutput\" \"yarn -s watch:bundle\" \"yarn -s start\"",
+        "watch": "concurrently --kill-others -n tsc,bundle -c red,yellow \"tsc -b -w --preserveWatchOutput\" \"yarn -s watch:bundle\"",
+        "watch:bundle": "theia build --watch --mode development",
+        "watch:compile": "tsc -b -w",
+        "download:plugins": "theia download:plugins"
```

Comment:
Some of the scripts behave different to the electron package.json. I think it would make sense to align at least `prepare`, `clean`, and `build`

Reply:
Right, I've addressed that in https://github.com/eclipse-cdt-cloud/cdt-cloud-blueprint/commit/3e4d5266524b3667c9300ab58d0929f1fff087df